webapp/src/shared/components/Input/Input.tsx

```diff
@@ -0,0 +1,41 @@
+import { Color } from '../../types';
+import Icon, { IconSize, IconVariant } from '../Icon/Icon';
+import './Input.css';
+
+export enum InputColor {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+
+type InputProps = {
+  color?: InputColor;
+  label?: string;
+  iconVariant?: IconVariant;
+  input?: string;
+};
+
+export default function Input({
+  color,
+  label,
+  iconVariant,
+  input,
+}: InputProps) {
+  return (
+    <div className="form-container subheading-bold">
+      {label && <label>{label}</label>}
```

Comment:
```suggestion
    <label className="form-container subheading-bold">
      {label}
```

suggestion: Here we could use the `label` tag directly as a wrapper to associate it with the `input`.

Reply:
Yes, it looks good. I'm changing it. And regarding the spacing, add it to #101 and I will also try to have a look